### PR TITLE
Use codecov to visualize coverage

### PR DESCRIPTION
**Pull request by syucream**
`Coverage` CI step doesn't work fine now, for e.g.:
```
...
curl: no URL specified!
curl: try 'curl --help' or 'curl --manual' for more information
```
https://app.circleci.com/pipelines/github/dmm-com/airone/112/workflows/f1d03d20-9004-4985-9214-f4ed111bb386/jobs/317

And also [codecov](https://about.codecov.io/) should help us to show visualized coverage without charges(its free on OSS repos). So I would like to replace existing coverage reporting stuff with codecovm like below:

image
https://codecov.io/gh/dmm-com/airone/tree/0540b9431f5323f9cd5bb0eda8b656d1940dd0ba

TODO

- [x] Investigate why some packages don't appear in coverage-xml result. => `source` directive on `.coveragerc`
- [x] Describe how to install it

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -21,7 +21,7 @@ whitelist_externals = rm
 
 [testenv:py35]
 # settings for selecting backend-db to use in test
-passenv = AIRONE_BACKEND_DB
+passenv = AIRONE_BACKEND_DB CODECOV_*
```
Comment (takano32) [inline tox.ini:25]: Please support py36.

py35 support will be droped.

File: .coveragerc [hunk 2]
```diff
@@ -1,12 +1,5 @@
 [run]
 branch = True
-source =
```
Comment (syucream) [inline .coveragerc:3]: It probably loses some modules on coverage 🤔

File: tox.ini (hunk 1 above)
Comment (syucream) [reply tox.ini:25]: I switched it!

**Review by takano32 — APPROVED** [review]
LGTM 👍

**Review by userlocalhost — APPROVED** [review]
LGTM

**Pull request merged**